Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy.swift

```diff
@@ -419,7 +419,7 @@ struct PathHierarchy {
     /// - Parameter name: The path component name of the tutorial overview (the file name without the file extension).
     /// - Returns: The new unique identifier that represent this tutorial overview.
     mutating func addTutorialOverview(name: String) -> ResolvedIdentifier {
-        return addNonSymbolChild(parent: tutorialOverviewContainer.identifier, name: name, kind: "technology")
+        return addNonSymbolChild(parent: tutorialOverviewContainer.identifier, name: name, kind: "tutorial-toc")
```

Comment:
Is this change safe to make? How does this interact with `DocumentationNode.Kind`?

Reply:
This doesn't have any relation to `DocumentationNode.Kind`. 

This value is only theoretically useful for link disambiguation but in practice it's not used because articles and tutorials have their own priorities when there's a collision and disambiguate by adding a "tutorials/" prefix for backwards compatibility.